src/safe_apps/tests/test_views.py

```diff
@@ -198,7 +198,7 @@ def test_should_cache_response(self):
             {
                 "url": safe_app_1.url,
                 "name": safe_app_1.name,
-                "icon_url": safe_app_1.icon_url,
+                "iconUrl": safe_app_1.icon_url,
                 "description": safe_app_1.description,
                 "networks": safe_app_1.networks,
                 "provider": None,
```

Comment:
Just for your reference, if you want to test stuff without camel case you can do `response.data` and you have the data without the camel case renderer applied.

I would also prefer if you use `response.json()` instead of `json.loads(response.content)` 😉

Reply:
Updated 👍 Thanks for the tip!